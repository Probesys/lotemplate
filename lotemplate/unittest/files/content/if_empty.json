{
  "empty_var": {
    "type": "text",
    "value": ""
  },
  "not_empty": {
    "type": "text",
    "value": "foo"
  },
  "space_var": {
    "type": "text",
    "value": "  \n   \t \t "
  }
}