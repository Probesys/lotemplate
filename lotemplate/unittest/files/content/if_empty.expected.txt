
Tests of if statements for empty strings

this content should be displayed ; 


 ; this one should be displayed

this content should be displayed because the var contains only spaces and tabs ; 

